archive_set_error(&a->archive, ARCHIVE_ERRNO_MISC,
	    "Using external lz4 program")